bin/copy

```diff
@@ -21,6 +21,7 @@ function ctt {
 
 mkdir -p ${TMP}/bin
 mkdir -p ${TMP}/lib/src/github.com/apg/patter/
+mkdir -p ${TMP}/lib/semver_bash/
```

Comment:
Maybe we should put this in `vendor/` instead? i.e. it's a lib we're vendoring, not one we've written.